libipl/p10/ipl0.C

```diff
@@ -547,6 +547,9 @@ static int ipl_updatehwmodel(void)
 	  	std::ofstream file(GENESIS_BOOT_FILE);
 	}
 
+	if(!boot_file_absent)
```

Comment:
Sorry i was wrong . As per @dcrowell77 document. this should invoked only for MPIPL, Graceful reboot from HB, reboot from phyp, webui.
we need to utlize the BOOTTIME_GUARD_INDICATOR for this purpose.
high level if (mpipl | present(BOOTTIME_GUARD_INDICATOR) )
       apply override .